libs/types/src/cdf/election-results-reporting/sample.ts

```diff
@@ -0,0 +1,305 @@
+import {
+  CountItemType,
+  ElectionReport,
+  ReportDetailLevel,
+  ReportingUnitType,
+  ResultsStatus,
+} from '.';
+import { ElectionType } from '../ballot-definition';
+
+export const electionResultsReport: ElectionReport = {
+  '@type': 'ElectionResults.ElectionReport',
+  IsTest: false,
+  Issuer: 'Sample County',
+  IssuerAbbreviation: 'SA',
+  VendorApplicationId: 'VotingWorks VxAdmin 1.0.2',
+  GeneratedDate: '2020-11-04T00:00:00Z',
+  SequenceStart: 1,
+  SequenceEnd: 1,
+  Status: ResultsStatus.UnofficialComplete,
+  GpUnit: [
+    {
+      '@id': 'sample-county',
+      '@type': 'ElectionResults.ReportingUnit',
+      Type: ReportingUnitType.County,
+      ComposingGpUnitIds: ['precinct-1', 'precinct-2'],
+    },
+    {
+      '@id': 'precinct-1',
+      '@type': 'ElectionResults.ReportingUnit',
+      Type: ReportingUnitType.Precinct,
+      ComposingGpUnitIds: ['precinct-1-1M', 'precinct-1-2F'],
+    },
+    {
+      '@id': 'precinct-1-1M',
+      '@type': 'ElectionResults.ReportingUnit',
+      Type: ReportingUnitType.SplitPrecinct,
+    },
```

Comment:
I defined these, and we could do vote totals at this granularity, but did not do so below.